packages/nextjs-kit/src/components/header.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Link from 'next/link';
+import { Headers } from '../index';
+
+interface HeaderProps {
+  headers: Headers[];
+}
+
+/**
+ * This is a Header component.
+ *
+ * @param {Props} headers holds the title and href of each route to be displayed
+ * @returns {React.JSX.Element}
+ */
+
+const Header: React.FC<HeaderProps> = ({ headers }: HeaderProps) => {
+  return (
+    <div className="my-0 pt-10 px-5 text-xl">
+      <nav>
+        <ul className="flex flex-row flex-wrap sm:flex-nowrap list-none justify-between max-w-screen-sm mx-auto">
+          {headers.map(item => (
+            <li
+              className={`${item.href === '/' ? 'mr-auto' : 'mx-4'}`}
```

Comment:
This works for  us, but I wonder if we can be more robust here. Maybe we can take in an "align" prop which defaults `right`, so that you can determine which side you want each menu item to be on.